### PR TITLE
add avatar and alert

### PR DESCRIPTION
**Pull request by ArjunJunna**
Have added avatar and alert components.

### CODE REVIEW

File: components/alert/alert.css [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+:root {
+  --border-color: rgb(0, 0, 0);
+  --danger-bg-color: #b80c00;
+  --warning-bg-color: rgb(233, 115, 18);
+  --success-bg-color: rgba(144, 221, 99, 0.671);
+  --danger-color: #F50F00;
+  --warning-color: rgb(230, 217, 198);
+  --success-color: rgb(31, 187, 44);
+}
```
Comment (fahd23) [inline components/alert/alert.css]: you can move this root element to component.css

File: components/alert/alert.css [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+.alert-component {
+  border: 1px solid var(--border-color);
+  margin: 4rem;
+  padding: 2.5rem;
+  border-radius: 0.7rem;
+}
+
+.alert__btn {
+  font-size: 2rem;
```
Comment (fahd23) [inline components/alert/alert.css:9]: Why do you use underscore here, is there any reason? if not, use the same naming convention

**Review by fahd23 — CHANGES_REQUESTED** [review]

File: components/alert/alert.css (hunk 1 above)
Comment (ArjunJunna) [reply components/alert/alert.css]: ok will do that

File: components/alert/alert.css (hunk 2 above)
Comment (ArjunJunna) [reply components/alert/alert.css:9]: Ya I tried to use BEM convention...Will try keeping it consistent thanks.

File: components/alert/alert.css (hunk 1 above)
Comment (ArjunJunna) [reply components/alert/alert.css]: just made changes. Have added the css variables to a designated folder and linked them.

**Comment by ShekharDhangar** [comment]
remove the inline styling of carbon iframe and put it in styles file

**Pull request merged**